docs/server/source/production-deployment-template/node-on-kubernetes.rst

```diff
@@ -128,7 +128,16 @@ Step 4.1: Vanilla NGINX
      the ConfigMap followed by ``-dep``. For example, if the value set in the
      ``ngx-instance-name`` is ``ngx-instance-0``, set  the
      ``spec.selector.app`` to ``ngx-instance-0-dep``.
-   
+     
+   * Set ``ngx-public-mdb-port`` to 27017, or the port number on which you
+     want to expose MongoDB service.
```

Comment:
It's not technically correct to write it as `ngx-public-mdb-port.port` because the YAML file hierarchy looks like this:

```yaml
spec:
  selector:
    app: ngx-instance-0-dep
  ports:
  - port: 27017
    targetPort: 27017
    name: ngx-public-mdb-port
    protocol: TCP
```

However, maybe it's clear what you mean, even though it's not 100% accurate. What do you think?